src/utils/guildRole.ts

```diff
@@ -135,3 +142,70 @@ export async function getGuildRoleByName(
   const roles = await getGuildRoles(env);
   return roles?.find((role) => role.name === roleName);
 }
+
+export async function mentionEachUserInMessage({
+  message,
+  userIds,
+  channelId,
+  env,
+}: {
+  message?: string;
+  userIds: string[];
+  channelId: number;
+  env: env;
+}) {
+  const batchSize = 5;
```

Comment:
This is because we can only send 5 messages without being rate-limited. If I increase the batchSize then the messages will start failing.

Reply:
I could not find it in the documentation because it says some of the routes have their own rate limiting and it was one of them. So upon trial and error, I found that it won't rate limit you for 5 API calls. But after that, it will ask you to wait for a few seconds which we can get from the headers.

Reply:
https://discord.com/developers/docs/topics/rate-limits#header-format
Is it possible to get it from the response

Reply:
If not then please add a comment there mentioning it

Reply:
<img width="1552" alt="Screenshot 2024-03-16 at 7 59 22 AM" src="https://github.com/Real-Dev-Squad/discord-slash-commands/assets/56365512/dd8731f3-50f8-44d5-a6d0-365e44d32d8a">


As you can see in the screenshot above. This is the response we get when we send a message in discord. It clearly says `x-ratelimit-limit:5` which implies it will apply rate limit after 5 API calls.

Reply:
Yes, so can't we take the value from there and set the rate limit, so if it increase or decrease, it will adjust accordingly

Reply:
Will be solving it here
- https://github.com/Real-Dev-Squad/discord-slash-commands/issues/208